convert/src/vertex_to_bd/models/bd_annotation_model.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+from typing import Any, List, Optional
+
+from pydantic import BaseModel
+
+
+class Size(BaseModel):
+    width: float
+    height: float
+
+
+class Entity(BaseModel):
+    id: str
+
+
+class Data(BaseModel):
+    entity: Entity
+
+
+class Obj(BaseModel):
+    id: int
+    type: str
+    bbox: List[float]
+    tokens: Optional[List[int]] = None
```

Comment:
Should it be Optional? Similar field below has empty list by default

Reply:
https://github.com/epam/badgerdoc/issues/170